miniapp/kernel/miniapp_larft_gemv.cpp

```diff
@@ -0,0 +1,230 @@
+                       stream);
+    };
+
+    [[maybe_unused]] auto kernel_GPU1 = [m, k, &vs, &ts](SizeType i, cublasHandle_t handle) {
+      gpulapack::larft_gemv1_notau(handle, m, k, vs(i).ptr(), vs(i).ld(), ts(i).ptr(), ts(i).ld());
+    };
+
+    [[maybe_unused]] auto post_kernel_GPU1 = [m, k, &taus, &ts](SizeType i, whip::stream_t stream) {
+      gpulapack::larft_gemv1_fixtau(k, taus(i).ptr(), 1, ts(i).ptr(), ts(i).ld(), stream);
+    };
+#endif
+    const double flop = ops<T>(n, k);
+
+    KernelRunner<backend> runner(opts.count, opts.nparallel);
+
+    for (SizeType run_index = 0; run_index < opts.nruns; ++run_index) {
+      double elapsed_time_pre = 0;
+      double elapsed_time_kernel = 0;
+      double elapsed_time_post = 0;
+      ts.setElements(el_t);
+
+      if constexpr (backend == Backend::MC) {
+        elapsed_time_kernel = runner.run(kernel_MC);
+      }
+#ifdef DLAF_WITH_CUDA
+      if constexpr (backend == Backend::GPU) {
+        switch (opts.kernel_id) {
+          case 0:
+            elapsed_time_kernel = runner.runHandle(kernel_GPU0);
+            elapsed_time_post = runner.runStream(copy_tau_in_t);
+            break;
+          case 1:
+            elapsed_time_pre = runner.runStream(copy_tau);
+            elapsed_time_kernel = runner.runHandle(kernel_GPU1);
+            elapsed_time_post = runner.runStream(post_kernel_GPU1);
+            break;
+          default:
+            std::cout << "Error: Non existing kernel id" << opts.kernel_id << std::endl;
+            elapsed_time_pre = -1;
+            elapsed_time_kernel = -1;
+            elapsed_time_post = -1;
+        }
+      }
+#endif
+
+      double elapsed_time = elapsed_time_pre + elapsed_time_kernel + elapsed_time_post;
+      const double gflops = flop / elapsed_time / 1e9;
+      const double gflops_kernel = flop / elapsed_time_kernel / 1e9;
+
+      std::cout << "[" << run_index << "]"
+                << " " << elapsed_time << "s"
+                << " " << gflops << "GFlop/s"
+                << " " << dlaf::internal::FormatShort{opts.type} << " ";
+      std::cout << m << " " << k << " " << ldv << " " << ldt << " " << opts.nparallel << " " << backend;
+      if (backend == Backend::GPU)
+        std::cout << " " << opts.kernel_id;
+
+      std::cout << " |"
+                << " PRE: " << elapsed_time_pre << "s"
+                << " KERNEL: " << elapsed_time_kernel << "s " << gflops_kernel << "GFlop/s"
+                << " POST: " << elapsed_time_post << "s" << std::endl;
+
+      if ((opts.do_check == dlaf::miniapp::CheckIterFreq::Last && run_index == (opts.nruns - 1)) ||
+          opts.do_check == dlaf::miniapp::CheckIterFreq::All) {
+        auto t = createTile<T, Device::CPU>(el_t, {k, k}, k);
+        MC_reference(m, k, v.ptr(), v.ld(), tau.ptr(), t.ptr(), t.ld());
+
+        auto error = ts.check(t);
+        if (error > k)
+          std::cout << "CHECK FAILED!!!: ";
```

Comment:
Is it this dramatic? :upside_down_face:
```suggestion
          std::cout << "CHECK FAILED!: ";
```
I see we are this dramatic in `miniapp_laset` as well, so either both change or neither changes... not a blocking comment.

Reply:
As dramatic as https://github.com/eth-cscs/DLA-Future/blob/master/miniapp/kernel/miniapp_laset.cpp#L130